addons/point_of_sale/static/src/app/store/product_configurator_popup/product_configurator_popup.xml

```diff
@@ -3,17 +3,17 @@
 
     <t t-name="point_of_sale.RadioProductAttribute">
         <div class="configurator_radio" t-ref="root">
-            <div class="d-flex flex-column gap-3">
+            <div class="d-flex flex-wrap gap-3">
                 <t t-foreach="values" t-as="value" t-key="value.id">
                     <div class="attribute-name-cell form-check">
-                        <input class="form-check-input" type="radio" t-model="state.attribute_value_ids" t-att-name="attribute.id"
+                        <input class="form-check-input radio-check" type="radio" t-model="state.attribute_value_ids" t-att-name="attribute.id"
                                 t-attf-id="{{ attribute.id }}_{{ value.id }}" t-att-value="value.id"/>
-                        <label class="form-check-label" t-att-name="value.name" t-attf-for="{{ attribute.id }}_{{ value.id }}">
-                            <span t-esc="value.name"/>
-                        </label>
-                        <div t-if="value.price_extra" class="price-extra-cell d-inline-block ms-3">
+                        <span t-esc="value.name"/>
+                        <div t-if="value.price_extra" class="price-extra-cell d-inline-block ms-2">
                             <span class="price_extra px-2 py-1 rounded-pill text-bg-info">
-                                + <t t-esc="env.utils.formatCurrency(value.price_extra)"/>
+                                <t t-if="0>value.price_extra"> - </t>
+                                <t t-else=""> + </t>
+                                <t t-esc="env.utils.formatCurrency(Math.abs(value.price_extra))"/>
```

Comment:
Since these are all just <t> elements, I'd suggest you create a method (maybe `getSigned(val)`) in the component and call it here with single <t t-out>.

```suggestion
                                <t t-out="getSigned(value.price_extra)" />
```